vcr/config.py

```diff
@@ -163,7 +165,12 @@ def _build_before_record_response(self, options):
         before_record_response = options.get(
             'before_record_response', self.before_record_response
         )
+        decode_compressed_response = options.get(
+            'decode_compressed_response', self.decode_compressed_response
+        )
         filter_functions = []
+        if decode_compressed_response:
+            filter_functions.append(filters.decode_response)
```

Comment:
Nit: I would put this filter BEFORE any of the explicitly passed in filters, so that those filters could access the decoded values if they needed to.